src/extensions/google-shopping-ads/release-notes/index.md

```diff
@@ -3,6 +3,9 @@ group: extensions
 title: Google Shopping ads Channel Release Notes
 ---
 
+{:.bs-callout .bs-callout-info}
```

Comment:
```suggestion
{:.bs-callout-warning}
```

Reply:
agreed to stay blue to be consistent with merchdocs banner

Reply:
same as on my previous comment